concepts/05 UI Components/Calendar/00 Getting Started with Calendar/15 Specify the Value Range.md

```diff
@@ -0,0 +1,74 @@
+Use the [min](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#max) properties to specify the range of available dates.
```

Comment:
How about we add informtation about [disabledDates](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#disabledDates) and rename the article? "Specify Available Dates"?

Reply:
Noted